app/js/components/LabOrdersList.jsx

```diff
@@ -81,11 +82,25 @@ const Cell = ({ columnName, value }) => {
 
 
 export class LabOrdersList extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.handleShowResultsEntryPage = this.handleShowResultsEntryPage.bind(this);
+  }
+
   componentDidMount() {
     const { dispatch } = this.props;
     dispatch(fetchLabOrders());
   }
 
+  handleShowResultsEntryPage(order) {
+    const { history } = this.props;
+    history.push({
+      pathname: "/LabResultEntry",
+      search: `?patientId=${order.patient.uuid}`,
```

Comment:
Do we need to add the patient here, or can we get it from the state, since it's part of the order?

Reply:
@mogoodrich that makes sense, I initially thought it would be okay to display the patientId on the url sort of